capabilities.tf.tmpl

```diff
@@ -0,0 +1,35 @@
+{{ range . -}}
```

Comment:
I assume this file is the same as what we have in our other application modules?

Reply:
Identical.